src/adapter/src/coord/sequencer.rs

```diff
@@ -2387,6 +2399,31 @@ impl<S: Append + 'static> Coordinator<S> {
         })
     }
 
+    async fn recent_timestamp(
```

Comment:
Can this get at least a vague doc comment describing the intent of the function and its asyncness?

Reply:
Added some docs.